tools/lsp/language/hover.rs

```diff
@@ -263,6 +267,12 @@ export component Test {
             get_tooltip(&mut dc, find_tk("the-ta := TA {", 11.into())),
             "```slint\ncomponent TA\n```",
         );
+        let target_path =
```

Comment:
I guess this should replace the following test, not?

Reply:
Yeap, obviously :-)

I accidentally squashed this into the previous PR that added the hover in the first place and then rebased that on top of master after merging the PR without this change, resolving the "conflicts" that got introduced.